tests.py

```diff
@@ -430,6 +438,121 @@ def test_data_with_gaps(self):
         self.assertTrue(os.path.isfile(expected_SFT_filepath))
 
 
+class TestTransientLineWriter(TestWriter):
+    label = "TestTransientLineWriter"
+    writer_class_to_test = pyfstat.make_sfts.TransientLineWriter
+    noiseSFT_detectors = "H1"
+
+    def test_multi_ifo_fails(self):
+        detectors = "H1,L1"
+        with pytest.raises(NotImplementedError):
+            self.writer_class_to_test(
+                label="test_noiseSFTs_noise_and_signal",
+                outdir=self.outdir,
+                duration=self.duration,
+                Tsft=self.Tsft,
+                tstart=self.tstart,
+                detectors=detectors,
+                sqrtSX=self.sqrtSX,
+                Band=0.5,
+                **self.signal_parameters,
+            )
+
+    def _check_maximum_power_consistency(self, writer, return_line_power=False):
```

Comment:
note to self: not checked the rest below here in detail yet

Reply:
@Rodrigo-Tenorio  The `_compare_corrected_powers` could be merged back into `test_cosi_scaling`, especially since it doesn't do the actual comparison itself (L538).

Apart from that, looks fine.